backtesting/backtesting.py

```diff
@@ -664,7 +664,8 @@ class _Broker:
     def __init__(self, *, data, cash, commission, margin,
                  trade_on_close, hedging, exclusive_orders, index):
         assert 0 < cash, f"cash should be >0, is {cash}"
-        assert 0 <= commission < .1, f"commission should be between 0-10%, is {commission}"
+        assert -.1 <= commission < .1, (f"commission should be between -10% "
+            "(e.g. market-maker's rebates) and 10% (fees), is {commission}")
```

Comment:
_This_ string literal should be the f-string. It's what's tripping the CI.

Reply:
yeah I put on the wrong position, sorry for polluting commit history. It should be squashed 😄